src/test/java/fi/aalto/cs/intellij/actions/EnablePluginsNotificationActionTest.java

```diff
@@ -0,0 +1,58 @@
+package fi.aalto.cs.intellij.actions;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.intellij.ide.plugins.IdeaPluginDescriptor;
+import com.intellij.notification.Notification;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.extensions.PluginId;
+import fi.aalto.cs.intellij.PluginsTestHelper;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class EnablePluginsNotificationActionTest extends PluginsTestHelper {
```

Comment:
Should only FTest classes extend `PluginTestHelper`. I see that methods like `getDummyPluginsListOfTwo` are used in this tests but, then again, `getDummyPluginsListOfTwo` does not use IntelliJ's test framework, right? So, perhaps, the class that provides helper methods like `getDummyPluginsListOfTwo` should be accessed through composition instead of inheritance.

Reply:
But this is a very minor issue. Absolutely no reason to refactor it now!